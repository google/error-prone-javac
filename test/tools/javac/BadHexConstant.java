/*
 * @test /nodynamiccopyright/
 * @bug 4049982
 * @summary Compiler permitted invalid hex literal.
 * @author turnidge
 *
 * @compile/fail/ref=BadHexConstant.out -XDrawDiagnostics  BadHexConstant.java
 */

public
class BadHexConstant {
    long i = 0xL;
}
