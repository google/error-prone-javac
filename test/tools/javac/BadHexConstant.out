BadHexConstant.java:12:14: compiler.err.invalid.hex.number
BadHexConstant.java:12:17: compiler.err.expected: token.identifier
2 errors
